features/borrower_views_deed.feature

```diff
@@ -0,0 +1,28 @@
+@borrower_views_deed
+
+Feature: Borrower Views the Deed
+  As a Borrower
+  I want to search for my mortgage document
+  So that I can view it
+
+  Background:
+    Given I have created a deed with two borrowers
```

Comment:
I would be tempted to rename this step in the feature file and step def file as at the moment the deed doesn't have any borrowers assigned to it.